pybloqs/htmlconv/wkhtmltox.py

```diff
@@ -1,8 +1,14 @@
+from __future__ import annotations
+
 import os
-from typing import Optional, Union
+from numbers import Number
+from typing import TYPE_CHECKING, Optional, Tuple, Union
 
 from pybloqs.htmlconv.html_converter import A4, PORTRAIT, HTMLConverter
 
+if TYPE_CHECKING:
+    from pybloqs import BaseBlock
```

Comment:
I think you have a bug here. BaseBlock is not defined when running without type annotations.

Maybe it works on latest python version and linters that ignore unknown type annotations, but I don't know if that works more widely.

Reply:
I think we are saved by the `from __future__ import annotations` [here](https://docs.python.org/3/library/typing.html#typing.TYPE_CHECKING), but that's not immediate. I'll do a quick survey to see what the state of the art is in other languages and then either use strings directly or continue with this.